examples/files.json

```diff
@@ -423,6 +423,7 @@
 		"misc_exporter_ply",
 		"misc_exporter_stl",
 		"misc_exporter_usdz",
+    "misc_exporter_usdz_compressed",
```

Comment:
Would you mind reverting this change? A new example isn't required for this feature.

Reply:
Understood, thanks!